dom/events/event-global.worker.js

```diff
@@ -0,0 +1,14 @@
+importScripts("/resources/testharness.js");
+test(t => {
+  let seen = false;
+  const event = new Event("hi");
+  assert_equals(self.event, undefined);
+  self.addEventListener("hi", t.step_func(e => {
```

Comment:
I think it's harmless, but the `t.step_func` isn't needed for synchronous tests like this one.

Reply:
Yes it is, what if the callback throws?

Reply:
The callback is invoked synchronously by `dispatchEvent` so the exception will reach the right place. Unless dispatchEvent suppresses exceptions and moves along to the next callback?

Reply:
That's not how event listeners work. They use "report the exception" which crashes the harness.

Reply:
Well there you go, I am just wrong :)